lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -23,6 +23,8 @@ import type {
 const markdown = new MarkdownIt('zero');
 markdown.enable(['heading', 'lheading']);
 
+const skippedRepositories = ['facebook/react-native'];
```

Comment:
let's add a comment why we've skip this, so nobody asks in future again 😅

Reply:
Smart 💡